README.md

```diff
@@ -42,6 +42,24 @@ jobs:
         github-token: "${{ secrets.GITHUB_TOKEN }}"
 ```
 
+If you want to use this action from a workflow file that doesn't run on the `pull_request` or `pull_request_target` events, use the `pull-request-number` input:
+
+```yaml
+name: Auto approve
+
+on:
+  workflow_dispatch
+
+jobs:
+  auto-approve:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: hmarr/auto-approve-action@v2
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        pull-request-number: 1234
```

Comment:
To avoid the hard-coded example number, perhaps this snippet could use [`workflow_dispatch` inputs](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#example-workflow-configuration) so we could change this to a dynamic value.

Alternatively, if you're looking to run this workflow after another Actions workflow that generates a pull request, [`workflow_run`](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_run) might be the trigger you're looking for?

Reply:
Done!

> Alternatively, if you're looking to run this workflow after another Actions workflow that generates a pull request, workflow_run might be the trigger you're looking for?

Thanks for the suggestion! Actually, I currently have my `automerge` workflow happening in response to the `pull_request` event, and it is successfully triggered after the workflow that generates a pull request. In case you're wondering _why_ I submitted this PR, I am effectively trying to make a "subroutine" that:

1) Generates a new version
2) Creates a PR to update the main branch with that version
3) Merges the PR so that the main branch is updated.

And I want that to happen all in one workflow, so that I can use https://github.com/softprops/turnstyle to prevent race conditions and merge conflicts. I'll use that subroutine in concert with https://github.com/convictional/trigger-workflow-and-wait to synchronously create a new version from any workflow that needs to do that. I'm hopeful that this will DRY up my workflows and simplify the handling of race conditions 😁